extensions/arc/src/ui/dashboards/postgres/postgresParameters.ts

```diff
@@ -381,15 +388,50 @@ export abstract class PostgresParametersPage extends DashboardPage {
 			return true */
 			this.saveButton!.enabled = true;
 			this.discardButton!.enabled = true;
+			this.collectChangedComponents(name);
 			return true;
 		}
 	}
 
 	protected createParameterComponents(engineSetting: EngineSettingsModel): ParametersModel {
 
-		// Container to hold input component and information bubble
-		const valueContainer = this.modelView.modelBuilder.flexContainer().withLayout({ alignItems: 'center' }).component();
+		// Can reset individual parameter
+		const resetParameterButton = this.modelView.modelBuilder.button().withProps({
+			iconPath: IconPathHelper.reset,
+			title: loc.resetToDefault,
+			width: '20px',
+			height: '20px',
+			enabled: true
+		}).component();
 
+		this.disposables.push(
+			resetParameterButton.onDidClick(async () => {
```

Comment:
This should clear out the component from the "changedComponents" list right?

Reply:
Right!